src/createServer.js

```diff
@@ -1,11 +1,246 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+  let users = [];
+  let expenses = [];
+
+  app.post('/users', express.json(), (req, res) => {
+    const { name } = req.body;
+    const newUser = {
+      id: Math.floor(Math.random() * 10),
+      name,
+    };
+
+    if (!name) {
+      res.sendStatus(400);
+
+      return;
+    }
+
+    users.push(newUser);
+
+    res.statusCode = 201;
+    res.send(newUser);
+  });
+
+  app.get('/users', express.json(), (req, res) => {
+    if (!users.length) {
+      res.send([]);
+
+      return;
+    }
```

Comment:
Why did you do this check?( I asked, because just interested about purpose, not saying that it is wrong, just trying to understand)